src/runtime/pkg/direct-volume/utils_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2022 Databricks Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package volume
+
+import (
+	"encoding/json"
+	"errors"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/kata-containers/kata-containers/src/runtime/pkg/uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAdd(t *testing.T) {
```

Comment:
Saw you added the check in the Add function -- can we add a test case to exercise this?

assert.Err(t, Add(volumePath, "{this actually isn't the json you wanted"}). ?